README.md

```diff
@@ -55,30 +55,49 @@ sudo apt update
 sudo apt install rdiff-backup
 ```
 
-### CentOS 7 (From COPR)
+### CentOS and RHEL 7 (From COPR)
 
 ```
-yum install yum-plugin-copr epel-release
-yum copr enable frankcrawford/rdiff-backup
-yum install rdiff-backup
+sudo yum install yum-plugin-copr epel-release
+sudo yum copr enable frankcrawford/rdiff-backup
+sudo yum install rdiff-backup
 ```
 
-### CentOS 8 (From COPR)
+### CentOS and RHEL 8 (From COPR)
 
 ```
-yum install dnf-plugins-core epel-release
-dnf copr enable frankcrawford/rdiff-backup
-yum install rdiff-backup
+sudo yum install dnf-plugins-core epel-release
+sudo dnf copr enable frankcrawford/rdiff-backup
+sudo yum install rdiff-backup
 ```
 
-### Debian (From PyPi)
+### Fedora 32+
+
+```
+sudo dnf install rdiff-backup
```

Comment:
More a question. Fedora 32+ already have v2.0.0 in default repo ? I tought rdiff-backup would need epel...
If we need epel, we should add the command line to install epel.

Reply:
EPEL is only for CentOS and RHEL, Fedora has much more packages and doesn't need it. And, yes, it's there, thanks to Frank!